### ISSUE TITLE
Popover arrow is missing/out of view with placement="auto | auto-bottom"

### ISSUE THREAD

**Issue by RajaJaganathan** [sandywalker/webui-popover#162]
Popover arrow is missing/out of view when placement set to auto or auto-bottom. Please find below simple to snippets to re-produce the issue. This issue happened when brower in maximized mode or browser width between 700px to 800px with below sample code.

webui popover plugin  - v1.2.6
OS - Mac OS
jQuery  - 2.1.4
Browser - Google Chrome Version 50.0.2661.94 (64-bit) 

Important: Make sure your browser in maximized mode.
```
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>webui popover</title>
	<link rel="stylesheet" href="dist/jquery.webui-popover.min.css" />
	<style>		
		body {
			width: 940px;
			margin: 0 auto;
		}
		
		.box {
			float: right;
			margin: 100px;
		}
		
	</style>
	<script src="jquery.min.js"></script>
	<script src="dist/jquery.webui-popover.min.js"></script>
	<script>
		$(function(){
			$('*[data-tooltip="popover"').webuiPopover('destroy').webuiPopover();
			$('*[data-tooltip="popover"').on('click',function(e){
				e.stopPropagation();
			});
		});
	</script>

</head>

<body>
	<div class="box">
		<button data-tooltip="popover" data-placement="auto-bottom" data-content="popover content is too great">
			Click me
		</button>	
	</div>
</body>

</html>
```

**Comment by sandywalker** [comment]
The bug is fixed in version 1.2.8 , please go and check!
Thank you for your bug report.

**Comment by RajaJaganathan** [comment]
Thanks @sandywalker. It is working fine now.
